src/Microsoft.VisualStudio.Editors/PropPages/DebugPropPage.vb

```diff
@@ -100,6 +101,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                     data = New PropertyControlData(VsProjPropId.VBPROJPROPID_RemoteDebugEnabled, "RemoteDebugEnabled", RemoteDebugEnabled, ControlDataFlags.PersistedInProjectUserFile)
                     datalist.Add(data)
 
+                    data = New PropertyControlData(VsProjPropId165.VBPROJPROPID_AuthenticationMode, "AuthenticationMode", AuthenticationMode, AddressOf SetAuthenticationMode, AddressOf GetAuthenticationMode, ControlDataFlags.PersistedInProjectUserFile, New Control() {AuthenticationModeLabel})
```

Comment:
Does this work in SDK-Style projects? Or does "AuthenticationMode" need to be added to a XAML rule file?

Reply:
SDK-style projects do not support remote debugging at all at the moment, so it is a moot point. We have a whole another user story to enable it.

Reply:
Oh yes, I forgot that the debug page isn't shared, though it's worth noting that basic remote debugging was added to the SDK-style debug page via a Remote Machine property.

Reply:
What basic remote debugging in .NET Core does, and what it uses for authentication? BTW, It will not work on Windows 10X with Windows authentication...

Reply:
It's literally just asking for a remote machine name at this point. There are screenshots in https://github.com/dotnet/project-system/pull/5784

Reply:
Question was more: What project system does with remote machine name? It has to pass it to debugger somehow...

Reply:
It passed (indirectly) to the debugger via VsDebugTargetInfo2.bstrRemoteMachine.